src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_index_pattern_wizard/components/step_time_field/step_time_field.js

```diff
@@ -81,12 +81,15 @@ export class StepTimeFieldComponent extends Component {
   }
 
   fetchTimeFields = async () => {
-    const { indexPatternsService, indexPattern } = this.props;
+    const { indexPatternsService, indexPattern: pattern } = this.props;
```

Comment:
i'm not sure, looks like it was added to avoid conflicting with the `const indexPattern` that is used below, but @ppisljar could probably explain more

Reply:
Peter says it is because `fieldsFetcher` is no longer available on `indexPatternsService`, just on each specific index pattern. So here it would probably be clearer to rename the prop `indexPatternName`